### PR TITLE
Docs to TTL feature

### PR DESCRIPTION
**Pull request by felipemorais**
Ready for "basket.js" english course.
Good luck to me.  :)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -182,35 +184,37 @@
...
-});
+	});
 </code></pre>
 
 
-<p><strong>basket.clear()</strong></p>
-<p>This will remove the keys associated at basket.js</p>
-<pre><code> basket.clear(); </code> </pre>
+<p><strong>basket.clear( expired )</strong></p>
+<p>This will remove the keys associated at basket.js. If expired is true, only expired keys will be removed.</p>
```
Comment (sindresorhus) [inline index.html]: at -> with

File: index.html [hunk 2]
```diff
@@ -83,6 +83,10 @@
 
 <pre><code>basket.require({ url: 'jquery.js' }, { url: 'underscore.js' }, { url: 'backbone.js' });</code></pre>
 
+<p>The HTML5 spec suggests a limit of 5 MB for amount of data written to localStorage, but browsers aren’t compelled to comply. So each browser can implement your own quota spaces. Basket.js will care exceeded quota removing oldest entries on load and when browser throws QUOTA_EXCEEDED_ERR error. Some browser like Opera can popup user asking about ilimited spaces at a given origin.</p>
+
+<p>basket.js will remove only your own entries. The spaces consumed by other apps in same origin will not be modified.</p>
```
Comment (sindresorhus) [inline index.html]: spaces -> space

File: index.html [hunk 3]
```diff
@@ -83,6 +83,10 @@
 
 <pre><code>basket.require({ url: 'jquery.js' }, { url: 'underscore.js' }, { url: 'backbone.js' });</code></pre>
 
+<p>The HTML5 spec suggests a limit of 5 MB for amount of data written to localStorage, but browsers aren’t compelled to comply. So each browser can implement your own quota spaces. Basket.js will care exceeded quota removing oldest entries on load and when browser throws QUOTA_EXCEEDED_ERR error. Some browser like Opera can popup user asking about ilimited spaces at a given origin.</p>
```
Comment (sindresorhus) [inline index.html]: The HTML5 spec suggests storage quota with the limit of 5MB for localStorage

Which means browsers can implement your own quota.

Comment (sindresorhus) [inline index.html]: Also what do you mean here:

> Some browsers like Opera can popup user asking about ilimited spaces at a given origin.

Does Opera ask the user to increase the localStorage quota?

Comment (felipemorais) [inline index.html]: Yes. Opera shows popup asking to user to increase quota when this exceed 1MB (and 2, 4, 8 ... ).

Comment (sindresorhus) [inline index.html]: k

"Some browsers like Opera will ask the user about increasing the quota when it exceeds a set threshold"

**Comment by sindresorhus** [comment]
Also, please squash your commits when done :)

https://github.com/addyosmani/aura/wiki/Git-Guide

**Pull request merged**